File: frontend/src/tipos/consulta-documento.ts
// Respuesta del endpoint /libro/:slug/consulta-documento/:numero
// Contiene los datos públicos de una persona o empresa obtenidos via PSE Perú.
export interface ConsultaDocumentoResponse {
  numero_documento: string;
  nombres: string;
  apellidos: string;
  nombre_razon: string;
  direccion: string;
}

File: backend/internal/model/dto/consulta_documento.go
package dto

// ConsultaDocumentoResponse contiene los datos públicos de una persona o empresa
// obtenidos del servicio PSE Perú, listos para devolver al frontend.
type ConsultaDocumentoResponse struct {
	NumeroDocumento string `json:"numero_documento"`
	Nombres         string `json:"nombres"`
	Apellidos       string `json:"apellidos"`
	NombreRazon     string `json:"nombre_razon"`
	Direccion       string `json:"direccion"`
}
